### ISSUE TITLE
Question - Is it possible to recover old blueprints?

### ISSUE THREAD

**Issue by Vicomt** [tsunamayo/Starship-EVO#4998]
we have a load of old .swbp (skywanderers) blueprint files. we've looked through the branches and none of the versions listed there will open them. is it possible to recover these blueprints? was there ever a version that opened and saved both .swbp and .mps files?

**Comment by Dwarf-LordPangolin** [comment]
My memory may be playing me false, but I seem to recall that you could do this at one time. [This post](https://github.com/tsunamayo/Starship-EVO/issues/954) seems to support that idea. But it looks like that would require a very old version, about two years old or older.

You _might_ be able to get such a version, however. There are directions [here](https://www.reddit.com/r/Steam/comments/611h5e/guide_how_to_download_older_versions_of_a_game_on/) on how to download an older version of a game from Steam; it requires a little digging, but it's doable. Looking at SteamDB, it seems that there are indeed some older, legacy versions left. One of those _might_ be able to open both formats.

**Comment by tsunamayo** [comment]
@Vicomt Hi, have you tried legacy_neovox?

**Comment by Vicomt** [comment]
@tsunamayo yeah, we tried all the available branches. none of the .swbp files showed up in any of them. I only placed them directly in  the bluepirnts folder though, as I believe the sub-folder structure only came in later. my memory might be hazy there though.

thanks for the info DLP, I'll have a look, although digging through all the versions might be a bit of a stretch ;)

**Comment by ProPeach** [comment]
I actually tried out that tool Pangolin posted, it's really cool. Sadly Steam Depot "only" goes back ~three years for sevo, so the earliest version you can get through it is 20w18a which isn't old enough to load .swbp files. I think you'd need a 2018 version for that at least, perhaps older
